Samples/PieChart/README.md

```diff
@@ -22,6 +29,6 @@ Slices are drawn using `PlotWedge`, like so:
 		</Panel>
 	</c:PlotData>
 
-UX expressions are used to color the wedges based on size, and an outer `Panel` is used to size the wedge based on size.
+UX expressions are used to color the wedges based on size, and an outer `Panel` is used to size the wedges, as the wedges occupy as much space as their parent element.
```

Comment:
Please start by explaining the problem that is solved by what you are talking about here. It needs some context to make any sense.